Packs/MicrosoftEntraID/README.md

```diff
@@ -0,0 +1,141 @@
+<~XSIAM>  
+
+<html>
+
+<h1>Microsoft Entra ID</h1>
+
+<details>
+    <summary><h2 style="display:inline-block">What does this pack do</h2></summary>
+
+<h3>Log Normalization - One Data Model</h3>
+<p>This pack support normalization of the below log categories of Microsoft Entra ID:</p>
+<ol>
```

Comment:
This should be a bulleted list.  Numbered lists are used when you have to do something in a particular order such as instructions.

Reply:
Changed it.